plugins/TagFix_Vatin.py

```diff
@@ -0,0 +1,74 @@
+#-*- coding: utf-8 -*-
+
+###########################################################################
+##                                                                       ##
+## This program is free software: you can redistribute it and/or modify  ##
+## it under the terms of the GNU General Public License as published by  ##
+## the Free Software Foundation, either version 3 of the License, or     ##
+## (at your option) any later version.                                   ##
+##                                                                       ##
+## This program is distributed in the hope that it will be useful,       ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of        ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+###########################################################################
+
+from plugins.Plugin import Plugin
+import re
+
+class TagFix_Vatin(Plugin):
+
+    def init(self, logger):
+        Plugin.init(self, logger)
+        self.errors[99999] = {"item": 9999, "level": 3, "tag": ["ref:vatin"], "desc": T_(u"Invalid ref:vatin")}
```

Comment:
Please be more explicit: `T_(u'Invalid value of tag "'ref:vatin"')`

Reply:
The `tag` cannot be `ref:vatin`. Pick tags from this list http://osmose.openstreetmap.fr/en/api/0.2/meta/tags